docs/getting-started.md

```diff
@@ -132,26 +132,31 @@ A subset of these nodes are deployed as a Slurm cluster. Unless instructed other
 
 ## What software is available on the IKIM cluster?
 
-The main entrypoint to the cluster is Slurm. On compute nodes, we aim to keep the environment as clean as possible, therefore only commonly used software packages are pre-installed and configured. At this moment the list includes:
+Short answer: Everything under the sun. You can install software yourself using either a [package manager](conda.pm) or build or run a [container](apptainer.md). Containers can be used e.g. to run a different operating system if you absolutely need to.
```

Comment:
`conda.pm` -> `conda.md`

Also, since slurm is a prerequisite for following the conda and apptainer guides, I would direct the reader to the slurm guide before anything else.

Reply:
fine with me.